src/Tests/Equinor.Procosys.Library.Query.Tests/GetAllDisciplines/GetAllDisciplinesQueryTests.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using Equinor.Procosys.Library.Query.GetAllDisciplines;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Equinor.Procosys.Library.Query.Tests.GetAllDisciplines
+{
+    [TestClass]
+    public class GetAllDisciplinesQueryTests
```

Comment:
add test asserting for ArgumentNullException when plant is null
Add test asserting for not null but empty list of classifications if not given